tests/schema/tests.py

```diff
@@ -178,6 +178,34 @@ def test_fk(self):
             self.fail("No FK constraint for author_id found")
 
     @skipUnlessDBFeature('supports_foreign_keys')
+    def test_fk_to_proxy(self):
+        "Tests that creating a FK to a proxy model creates database constraints."
```

Comment:
I agree with you but I used this sentence structure based on the two test method wrapping this one.